src/client/components/Container.tsx

```diff
@@ -30,9 +30,13 @@ const noPaddingStyles = css`
   }
 `;
 
-const sideBorderColors = (color: string) => css`
-  border-left-color: ${color} !important;
-  border-right-color: ${color};
+const sideBorderStyles = (color = neutral[86]) => css`
+  > div {
+    ${from.tablet} {
+      border-left: 1px solid ${color};
+      border-right: 1px solid ${color};
+    }
+  }
```

Comment:
Fixes border alignment on the Container component used by the navigation bar, footer, and layouts.